src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,13 @@
-const fizzBuzz = number => {};
-
+const fizzBuzz = number => {
+   if (number % 5 === 0 && number % 3 === 0) {
```

Comment:
Good stuff! You can shorten this a little as the lowest common multiple of 5 and 3 is 15, so `number % 15 === 0` would also work.